openghg/util/_user.py

```diff
@@ -153,3 +146,30 @@ def read_local_config() -> Dict:
 
     config: Dict = toml.loads(config_path.read_text())
     return config
+
+
+def check_config() -> None:
+    """Check that the user config file is valid and the paths
+    given in it exist.
+
+    Returns:
+        bool
+    """
+    config_path = get_user_config_path()
+
+    if not config_path.exists():
+        logger.warning("Configuration file does not exist. Please create it by running openghg --quickstart.")
+
+    config = read_local_config()
+    uid = config["user_id"]
+    object_stores = config["object_store"]
+
+    try:
+        uuid.UUID(uid, version=4)
+    except ValueError:
+        logger.exception("Invalid user ID. Please re-run quickstart to setup a valid config file.")
+        raise
```

Comment:
Was this meant to raise a specific error here?

Reply:
Because it's in the except of a ValueError, it's so you can do something before re-raising the exception.

Reply:
Ah that's useful to know and I've learnt something. Thanks for explaining.